### PR TITLE
MOSIP-27669 Renaming the IDP scripts and Readme file to Esignet

### PR DESCRIPTION
**Pull request by Sanjayss04**

### CODE REVIEW

File: Esignet/README.md [hunk 1]
```diff
@@ -20,12 +20,12 @@
...
 
-* Create OIDC Client in MOSIP Authentication System (Setup) : This thread contains a JSR223 sampler(Generate Key Pair) from which will get a public-private key pair. The public key generated will be used in the OIDC client api to generate client id's  which will be registered for both IDA and IDP. The private key generated from the sampler will be used in another JSR223 sampler(Generate Client Assertion) present in the OIDC Token (Execution). Generated client id's and there respective private key will be stored in a file which will be used further in the required api's.
+* Create OIDC Client in MOSIP Authentication System (Setup) : This thread contains a JSR223 sampler(Generate Key Pair) from which will get a public-private key pair. The public key generated will be used in the OIDC client api to generate client id's  which will be registered for both IDA and Esignet. The private key generated from the sampler will be used in another JSR223 sampler(Generate Client Assertion) present in the OIDC Token (Execution). Generated client id's and there respective private key will be stored in a file which will be used further in the required api's.
```
Comment (vishwa-vyom) [inline Esignet/README.md]: It should writen as eSignet, make this change everywhere in readme file and base folder rename to esignet.

https://docs.esignet.io/

File: Esignet/scripts/Esignet_Test_Script.jmx [hunk 2]
```diff
@@ -1709,6 +1923,15 @@ vars.put(&quot;publicKey&quot;, publicKey);
             <stringProp name="RegexExtractor.match_number">1</stringProp>
           </RegexExtractor>
           <hashTree/>
+          <RegexExtractor guiclass="RegexExtractorGui" testclass="RegexExtractor" testname="Extract Captcha" enabled="true">
```
Comment (vishwa-vyom) [inline Esignet/scripts/Esignet_Test_Script.jmx]: For capcha, what are we doing ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: Esignet/README.md (hunk 1 above)
Comment (Sanjayss04) [reply Esignet/README.md]: Done

File: Esignet/scripts/Esignet_Test_Script.jmx (hunk 2 above)
Comment (Sanjayss04) [reply Esignet/scripts/Esignet_Test_Script.jmx]: We are getting the captcha value which is generated from the preparation thread group response. And we are storing the value in a file and using the same in the execution.

Comment (vishwa-vyom) [reply Esignet/scripts/Esignet_Test_Script.jmx]: Are you sure this works ? Was the captcha enabled when the script were run ?

Comment (Sanjayss04) [reply Esignet/scripts/Esignet_Test_Script.jmx]: Captcha is disabled. Have updated the endpoint will captcha field as empty.

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**